account_bank_statement_import_paypal/README.rst

```diff
@@ -0,0 +1,41 @@
+Import Paypal Bank Statements
+=============================
+
+This module allows you to import the Paypal CSV files in Odoo as bank statements.
```

Comment:
You might want to describe which kind of PayPal CSV file exports are accepted. I can download 3 types (all activity, balance affecting payments or completed payments) from PayPal of which sometimes only the balance affecting payments imports properly.